app/styles/components/_buttons.scss

```diff
@@ -1,17 +1,17 @@
 .btn {
   &.on-white {
-    background-color: $blue_light;
-    color: $navy;
+    background-color: #08987f;
+    color: $white;
```

Comment:
What do you think about swapping the hover styles with the values you've set on lines 3 and 4, so the lighter version is the default, and the hover version is the darker version? That would help keep the default state closer to the brand green.